### DISCUSSION TITLE
History links to previous test report and test case

### DISCUSSION THREAD

**Discussion by lunmag** [allure-framework/allure2#2100] [Questions & Support] [answered]
Hi!

Do one need to do some coding to make the history of test case results provide links to the respective test result/report.
When just trying the "copy history folder"-method locally I _only_ get the status of the older test runs, which is nice of course :-) .
When I look at the Demo project ( https://allure-framework.github.io/allure-demo/7/ ) the test cases have individual links.
Thanks for a great reporting tool!
Searched for "history link" in the forum...

/M

**Comment by Hydrair** [comment]
I'm also trying to achieve this. Cloning and creating reports with the demo repo is also missing the links.

**Comment by baev** [comment, marked as answer]
https://github.com/orgs/allure-framework/discussions/2249#discussioncomment-7844167

You need to provide `reportUrl` via `executor.json` to make links clickable
